.codeclimate.yml

```diff
@@ -44,17 +44,6 @@ plugins:
      - decidim-comments/app/assets/javascripts/decidim/comments/bundle.js.map
      - .rubocop.yml
 
-  grep:
-    enabled: true
-
-    config:
-      patterns:
-        no-trailing-whitespace:
-          pattern: \s*$
-          annotation: "Don't leave trailing whitespace"
-          severity: minor
-          categories: Style
```

Comment:
This ensures that we have no trailing whitespace globally (not only for ruby files). I guess we no longer want that?

Reply:
Well, surprisingly it was just blowing up on code climate so I didn't know what else to do xD. It looks like I can't navigate the history, but it was an obscure ruby error: `gsub can't be called on nil` or something like that.

Reply:
Can you add `decidim-design/**/*` to the exclusions of that check? If you have no time or it doesn't work it's ok, I can investigate what happened at a later time.